### PR TITLE
Avoid a second query for the current color-scheme

### PR DESCRIPTION
**Pull request by WhyNotHugo**
Telegram listens for a signal that indicates when the color-scheme
changes. The signal itself includes the new value, but Telegram
currently queries for the value immediately after getting the signal.
This second round-trip is unnecessary, since the signal itself contains
the same information.

This changeset avoids this follow-up query, and drops the now-unused
`Setter`.

### CODE REVIEW

**Comment by WhyNotHugo** [comment]
Ugh... CI won't work because my username is uppercase. :facepalm: 

I'm sure I've seen this before and working around it was an ugly hack :(

File: Telegram/SourceFiles/platform/linux/specific_linux.cpp [hunk 1]
```diff
@@ -396,7 +390,11 @@ std::optional<bool> IsDarkMode() {
 				const Glib::VariantBase &value) {
 				if (group == "org.freedesktop.appearance"
 					&& key == "color-scheme") {
-					Setter();
+					const auto ivalue = base::Platform::GlibVariantCast<uint>(value);
```
Comment (ilya-fedin) [inline Telegram/SourceFiles/platform/linux/specific_linux.cpp:393]: GlibVariantCast can throw an exception. You should use try-catch.

**Comment by john-preston** [comment]
Thanks!

**Comment by john-preston** [comment]
In case you're interested, we have a Telegram group for code / contributions discussion: https://t.me/tdesktop_contributions

**Pull request merged**